conf/amavisd.conf.in

```diff
@@ -267,7 +267,7 @@ $archive_quarantine_to = undef;
 
 $banned_filename_re = new_RE(
   # banned extension - basic
-  %%uncomment VAR:zimbraMtaBlockedExtension%%qr'.\.(%%list VAR:zimbraMtaBlockedExtension |%%)$'i,
+  %%uncomment VAR:zimbraMtaBlockedExtension%%qr{.\.(?:%%list VAR:zimbraMtaBlockedExtension |%%)\s*$}i,
```

Comment:
Done a quick search for ?: found it means that the group is matched but is not captured for back-referencing.
Is this what you want to achieve @AMSrivastava 
https://stackoverflow.com/questions/36524507/notation-in-regular-expression

Reply:
As @log2akshat wrote does using `(?:)` avoid that the captured string is assigned to a variable. This is a common optimization in Perl (used a lot in SpamAssassin rules) which I added since I was changing that line anyway.

I changed the `'` to curly brackets since it is also more common in Perl code (and better supported in code highlighters) and IMO makes the line easier to read.

Reply:
Thanks for the clarification!